crypto.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import re
+
+from decorators      import is_command
+from crypto_data     import crypto_data
+
+def get_key(value):
+    for key, value in crypto_data.items():
+        if value == value:
+            return key
+
+    return "BTC"
```

Comment:
If not found, this should either except or return None, which the calling function should handle.

Reply:
Yeah, I did a lazy pass here to return BTC to not handle the exception. Definitely can fix.